src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java

```diff
@@ -47,9 +53,9 @@ public class ComponentAssemblyLine extends GT_MetaTileEntity_LongPowerUsageBase<
                     STRUCTURE_PIECE_MAIN,
                     new String[][] {
                             { "         ", "   III   ", " HHI~IHH ", "HH III HH", "H       H", "H       H", "H  JJJ  H",
-                                    "H  N N  H", "H  N N  H", "HHHHHHHHH" },
-                            { "         ", " EHHHHHE ", "E       E", "H       H", "A       A", "A       A", "A  HHH  A",
-                                    "A       A", "A       A", "MHHHHHHHM" },
+                                    "H  JJJ  H", "H  N N  H", "HHHHHHHHH" },
+                            { "         ", " EHHHHHE ", "E       E", "H       H", "A       A", "A       A", "A       A",
+                                    "A  HHH  A", "A       A", "MHHHHHHHM" },
```

Comment:
it will break existed coal setups

Reply:
yes, but the change is very minor. plus, the input hatches that are already in their positions before the update will still keep their same positions. not much change is going to be needed.